concepticondata/conceptlists/Janschewitz-2008-460.tsv

```diff
@@ -0,0 +1,461 @@
+Janschewitz-2008-460-391	391	clock	1323	CLOCK	rel neu	5.99	6.21	01.03	01.01	05.04	1.32	08.08
+Janschewitz-2008-460-392	392	closet	3357	CUPBOARD	rel neu	5.86	5.74	01.01	1.14	4.94	1.52	7.19
+Janschewitz-2008-460-393	393	cloth	1616	CLOTH	rel neu	4.13	4.68	01.01	01.08	05.08	1.34	7.14
+Janschewitz-2008-460-394	394	cook	1100	COOK (SOMETHING)	rel neu	6.16	6.65	01.06	01.01	5.65	2.61	6.56
+Janschewitz-2008-460-395	395	couch	3099	SOFA	rel neu	5.94	6.27	1	01.01	5.1	1.69	08.01
+Janschewitz-2008-460-396	396	cupboard	3357	CUPBOARD	rel neu	4.34	4.94	01.04	01.03	05.01	1.31	7.47
+Janschewitz-2008-460-397	397	curtains	2621	CURTAIN	rel neu	4.56	4.91	01.01	01.01	5	1.29	08.08
+Janschewitz-2008-460-398	398	den	2949	LAIR	rel neu	3.39	4.25	01.01	01.03	05.04	1.47	5.26
+Janschewitz-2008-460-399	399	desk	1775	TABLE	rel neu	5.71	06.03	01.01	1	5.13	1.32	8.14
+Janschewitz-2008-460-400	400	dish	481	DISH	rel neu	5.75	06.04	01.01	01.01	4.99	1.36	7.65
+Janschewitz-2008-460-401	401	dresser			rel neu	4.68	4.77	01.01	01.01	4.97	1.31	08.01
+Janschewitz-2008-460-402	402	dryer			rel neu	5.55	5.75	01.01	01.01	05.05	1.38	7.79
+Janschewitz-2008-460-403	403	fireplace	185	FIREPLACE	rel neu	4.14	4.64	01.01	01.04	5.38	2.16	8.32
+Janschewitz-2008-460-404	404	freezer			rel neu	5.62	5.71	01.01	01.01	5	1.39	7.69
+Janschewitz-2008-460-405	405	furnace			rel neu	2.47	3.29	01.01	01.03	05.04	1.46	6.17
+Janschewitz-2008-460-406	406	furniture			rel neu	5.28	5.82	01.01	1	05.05	1.45	7.69
+Janschewitz-2008-460-407	407	glass			rel neu	5.71	06.04	1	1	05.05	1.32	7.38
+Janschewitz-2008-460-408	408	hammer	1481	HAMMER	rel neu	3.97	4.73	01.03	01.05	4.95	1.7	8.38
+Janschewitz-2008-460-409	409	iron	621	IRON	rel neu	4.2	4.74	01.01	01.03	05.03	1.28	7.49
+Janschewitz-2008-460-410	410	jar	1462	POT	rel neu	4.16	4.73	01.01	01.01	05.01	1.29	08.04
+Janschewitz-2008-460-411	411	kettle	854	KETTLE	rel neu	3.25	3.83	01.01	01.01	05.04	1.38	6.93
+Janschewitz-2008-460-412	412	ladder	1464	LADDER	rel neu	3.83	4.45	01.01	01.01	05.03	1.48	8.13
+Janschewitz-2008-460-413	413	lamp	625	LAMP	rel neu	05.07	5.42	01.01	1	5.13	1.32	8.12
+Janschewitz-2008-460-414	414	lawn	3990	LAWN	rel neu	4.1	5	01.01	01.01	5.13	1.48	7.45
+Janschewitz-2008-460-415	415	mantel			rel neu	2.76	3.5	01.01	1.16	05.04	1.2	6.2
+Janschewitz-2008-460-416	416	microwave			rel neu	5.95	6.34	01.04	01.01	5.1	1.75	8.19
+Janschewitz-2008-460-417	417	mirror	1171	MIRROR	rel neu	5.81	6.14	1	01.01	5.13	1.57	7.88
+Janschewitz-2008-460-418	418	mixer			rel neu	3.88	4.61	01.01	01.06	5.14	1.45	5.4
+Janschewitz-2008-460-419	419	mop	3120	MOP (SOMETHING)	rel neu	3.9	4.6	01.01	01.04	4.96	1.42	7.96
+Janschewitz-2008-460-420	420	paint	2001	PAINTING	rel neu	4.51	5.34	01.03	01.01	5.34	1.96	7.27
+Janschewitz-2008-460-421	421	pan	31	PAN	rel neu	4.97	05.04	01.01	01.01	5	1.32	7.7
+Janschewitz-2008-460-422	422	pantry			rel neu	3.58	4.29	01.01	01.08	5.23	1.65	6.81
+Janschewitz-2008-460-423	423	patio			rel neu	4.12	4.75	01.01	01.01	05.08	1.56	7.48
```

Comment:
[YARD](https://concepticon.clld.org/parameters/736)?

Reply:
We don't have a mapping with the English gloss 'patio' yet.